addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -266,6 +274,7 @@ def _l10n_in_edi_ewaybill_irn_post_invoice_edi(self, invoices):
             )
 
             invoices.message_post(body=body)
+            self._l10n_in_edi_ewaybill_handle_zero_distance_alert_if_present(invoices, response)
```

Comment:
Can we change invoices, to invoice at least, because I suppose we are just talking about one invoice anyways.

Reply:
@jco-odoo better to do it in master. I have already started with the refactor. I don't want any other e waybill PR. Other than what we are about to merge.